### PR TITLE
Translate tStudent.C to python

### PR DESCRIPTION
**Pull request by juanfe**

### CODE REVIEW

File: tutorials/math/tStudent.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+n = 100
+a = -5.
+b = 5.
+pdf = TF1("pdf", "ROOT::Math::tdistribution_pdf(x,3.0)", a, b)
+cum = TF1("cum", "ROOT::Math::tdistribution_cdf(x,3.0)", a, b)
+
+quant = TH1D("quant", "", 9, 0, 0.9)
+
+for i in range(1, 10):
+    quant.Fill((i-0.5)/10.0, ROOT.Math.tdistribution_quantile((1.0 * i) / 10,
```
Comment (dpiparo) [inline tutorials/math/tStudent.py]: Perhaps this is in the original version but why  having divisions around when we know they are poison for cpus wrt multiplications?

File: tutorials/math/tStudent.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                                                              3.0))
+
+xx = []
+xx.append(-1.5)
+for i in range(1, 9):
+    xx.append(quant.GetBinContent(i))
```
Comment (dpiparo) [inline tutorials/math/tStudent.py]: Something to make this nice translation even more Pythonic: why not list comprehensions?

File: tutorials/math/tStudent.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+xx.append(1.5)
+pdfq = []
+for i in range(9):
+    nbin = int(n * (xx[i+1] - xx[i]) / 3.0 + 1.0)
+    name = "pdf"
+    name += str(i)
```
Comment (dpiparo) [inline tutorials/math/tStudent.py]: string formatting?

File: tutorials/math/tStudent.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+quant.Draw()
+quant.SetLineWidth(2)
+quant.SetLineColor(kBlue)
+quant.SetStats(0)
+Canvas.cd(4)
+pdfq[0].SetTitle("Student t & its quantiles")
+pdf.SetTitle("")
+pdf.Draw()
+pdfq[0].SetTitle("Student t & its quantiles")
+for i in range(9):
```
Comment (dpiparo) [inline tutorials/math/tStudent.py]: Why 9? Perhaps we can use a standard python loop w/o indices?

File: tutorials/math/tStudent.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    pdfq[i].SetStats(0)
+    pdfq[i].SetFillColor(i+1)
+    pdfq[i].Draw("same")
+Canvas.Modified()
+Canvas.cd()
```
Comment (dpiparo) [inline tutorials/math/tStudent.py]: Why this last cd?

File: tutorials/math/tStudent.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+## \macro_code
+##
+## \author Juan Fernando Jaramillo Botero
+
+from ROOT import TH1D, TF1, TCanvas, kRed, kBlue
+import ROOT
+
+
+# this is the way to force load of MathMore in Cling
+ROOT.Math.MathMoreLibrary.Load()
```
Comment (dpiparo) [inline tutorials/math/tStudent.py]: Is this really needed or autoloading kicks in automatically?

File: tutorials/math/tStudent.py (hunk 1 above)
Comment (juanfe) [reply tutorials/math/tStudent.py]: I try, but I not sure how.

File: tutorials/math/tStudent.py (hunk 6 above)
Comment (juanfe) [reply tutorials/math/tStudent.py]: The line:
quant.Fill((i-0.5)/10.0, ROOT.Math.tdistribution_quantile((1.0 * i) / 10, 3.0))
requires this Load.

File: tutorials/math/tStudent.py (hunk 4 above)
Comment (juanfe) [reply tutorials/math/tStudent.py]: I did, but after that in the original code he use the i index to set the color, I introduce an index to do that. Any alternative suggestion?

File: tutorials/math/tStudent.py (hunk 1 above)
Comment (etejedor) [reply tutorials/math/tStudent.py]: Perhaps @dpiparo refers to converting the /10.0 and /10 divisions into *0.1 multiplications?

**Comment by etejedor** [comment]
Hi @juanfe, thank you for submitting this tutorial. A general comment I have is: for your tutorial to look better in notebook form, can you split it in code blocks with their corresponding comments, which then will be translated to notebook cells? E.g.:
```python
# Here I explain what the following lines do, this will be converted to a text cell
Here I put
my code lines.
This will be all
in the same code cell

# Here I start another text cell
Here is the code 
for another
code cell
```

**Comment by etejedor** [comment]
Hi @juanfe is this still something you would like to merge? If so, please address the comments of @dpiparo and mine and we will be good to go :smiley:

**Comment by etejedor** [comment]
Hi @juanfe , thank you for addressing @dpiparo's comments. Please also make the tutorial more notebook-compliant as I explain in my comment and we will be good to go!

**Comment by etejedor** [comment]
@phsft-bot build

**Comment by etejedor** [comment]
@dpiparo is this PR ok for you too? If so I will squash and merge once the builds confirm

**Comment by dpiparo** [comment]
@etejedor lgtm!

**Comment by etejedor** [comment]
Hi @juanfe, I realized there is one small thing to add. In the following line of `CMakeLists.txt`

https://github.com/root-project/root/blob/master/tutorials/CMakeLists.txt#L383

we need to make sure the `tStudent.py` tutorial is vetoed if `libMathMore` is not there, just like e.g. `Bessel.py` is, by adding this line:
```
list(APPEND pyveto math/tStudent.py)
```

File: tutorials/CMakeLists.txt [hunk 7]
```diff
@@ -375,6 +375,8 @@ if(ROOT_python_FOUND)
              )
   if(NOT ROOT_mathmore_FOUND)
     list(APPEND pyveto math/Legendre.py)
+    list(APPEND pyveto math/Bessel.py)
```
Comment (etejedor) [inline tutorials/CMakeLists.txt]: Hi @juanfe please rebase with master, `Bessel.py` is already vetoed in master, your commit should only include `tStudent.py`, thanks!

Comment (juanfe) [reply tutorials/CMakeLists.txt]: Done.

**Pull request merged**